tests/apollo/test_skvbc_restart_recovery.py

```diff
@@ -120,13 +125,6 @@ async def _restarting_replica_during_system_is_in_view_change(self, bft_network,
         next_primary = 1
         view = 0
 
-        # uncomment for live tracking of log messages from the test
```

Comment:
It will be good to remove this block from all the tests now that we have extracted it to be general for the entire test suite. This is much better for reducing the noise.

Reply:
Done